src/supertux/main.cpp

```diff
@@ -366,8 +366,10 @@ Main::launch_game()
 
   timelog("audio");
   SoundManager sound_manager;
-  sound_manager.enable_sound(g_config->sound_enabled);
-  sound_manager.enable_music(g_config->music_enabled);
+  sound_manager.enable_sound(g_config->sound_volume > 0 ? true : false);
+  sound_manager.enable_music(g_config->music_volume > 0 ? true : false);
```

Comment:
How is it a fault in OpenAL that a volume of 0 is equivalent to mute?  This would seem to be the behavior of any rational program.  I also question if it is advisable to add even more options to the settings dialog-it already is pretty long as it is.  While allowing the user to short-circut the selector to 0/mute might be useful, having music on/off and music volume as separate line items seems redundant and a waste of space.